srcpkgs/python3-bidict/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'python3-bidict'
+pkgname=python3-bidict
+version=0.21.3
+revision=1
+wrksrc="bidict-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3"
+checkdepends="python3-hypothesis python3-py-cpuinfo python3-pytest python3-Sphinx"
+short_desc="Bidirectional mapping library for Python"
+maintainer="DragonGhost7 <darkiridiumghost@gmail.com>"
+license="MPL-2.0"
+homepage="https://github.com/jab/bidict"
+distfiles="${PYPI_SITE}/b/bidict/${wrksrc}.tar.gz"
+checksum=d50bd81fae75e34198ffc94979a0eb0939ff9adb3ef32bcc93a913d8b3e3ed1d
+
+post_extract() {
+	sed -e '/--benchmark-columns=mean,stddev,outliers/,+3d' -i pytest.ini
+rm tests/test_benchmark.py
+# void linux doesnt have pytest benchmark tool
+	}
+
+do_check() {
+	python3 -m pytest tests
+}
```

Comment:
```suggestion
make_check_target=tests
```
but is it really required?

Reply:
yes because otherwise it tests coverage too and we dont really need that